.gitlab-ci.yml

```diff
@@ -325,6 +336,21 @@ pkg:nix:deploy:
     - master
     - /^v.*\..*$/
 
+pkg:nix:deploy:channel:
+  <<: *deploy-template
+  environment:
+    name: cachix
+    url: https://coq.cachix.org
+  only:
+    variables:
+      - $CACHIX_DEPLOYMENT_KEY
+  dependencies:
+    - pkg:nix:deploy
+  script:
+    - echo "$CACHIX_DEPLOYMENT_KEY" | tr -d '\r' | ssh-add - > /dev/null
+    - git fetch --unshallow
```

Comment:
Why not override GIT_DEPTH?

Reply:
That's what I wanted to do at first but nowhere in the documentation it says what value to put if you want no limit (I suppose empty would work but this is unspecified). As far as I know, we can't unset a variable that is defined globally...